src/EditorFeatures/CSharp/Formatting/CSharpEditorFormattingService.cs

```diff
@@ -33,16 +34,12 @@ internal partial class CSharpEditorFormattingService : IEditorFormattingService
         // All the characters that might potentially trigger formatting when typed
         private readonly char[] _supportedChars = ";{}#nte:)".ToCharArray();
 
-#nullable enable
```

Comment:
Nit: nullable enable for the entire file? Unless it is causing lot of changes...

Reply:
Well, the rest of the file is large and unrelated to this. I'm also not sure I know some of it well enough to be comfortable annotating.

Reply:
Looking at this now, but since it's a fairly different change I'll do that as a separate PR.

Reply:
Done: #40339